src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -1559,7 +1559,9 @@ private void DoMiscValidation()
 
             void validateParamsType(BindingDiagnosticBag diagnostics)
             {
-                var collectionTypeKind = ConversionsBase.GetCollectionExpressionTypeKind(DeclaringCompilation, Type, out TypeWithAnnotations elementTypeWithAnnotations);
+                var syntax = ParameterSyntax;
+                var binder = GetDefaultParameterValueBinder(syntax).WithContainingMemberOrLambda(ContainingSymbol); // this binder is good for our purpose
```

Comment:
I know this is an existing comment, but... _why_ is this binder good for our purpose?

Reply:
Not sure, but I don't think this PR affects that choice of binder.